src/features/speciesfeature.jl

```diff
@@ -15,9 +17,10 @@ Construct a feature object that encodes features associated with individual atom
 - `length::Int`: length of encoded vector
 - `encodable_elements::Vector{String}`: list of elements (by symbol) that can be encoded by this feature
 """
-struct SpeciesFeatureDescriptor <: AbstractAtomFeatureDescriptor
+struct SpeciesFeatureDescriptor{A} <: AbstractAtomFeatureDescriptor
     name::String
     length::Integer
+    compute_f::Any
```

Comment:
Is `compute_f` the function mapped to `A`? Also, adding a line in the docstring might be helpful 😅

Reply:
haha yeah this part still needs to be fleshed out a bit; I've gotten sidetracked by some of the other changes